doc/Advanced-configuration.md

```diff
@@ -256,26 +254,26 @@ See [Database backends configuration](./advanced-configuration/database-backends
     * **Syntax:** `{pgsql_users_number_estimate, false | true}`
     * **Default:** `false`
 
-* **odbc_keepalive_interval** (local)
+* **rdbms_keepalive_interval** (local)
     * **Description:** When enabled, will send `SELECT 1` query through every DB connection at given interval to keep them open.
     This option should be used to ensure that database connections are restarted after they became broken (e.g. due to a database restart or a load balancer dropping connections).
     Currently, not every network related error returned from a database driver to a regular query will imply a connection restart.
-    * **Syntax:** `{odbc_keepalive_interval, IntervalSeconds}.`
-    * **Example:** `{odbc_keepalive_interval, 30}.`
+    * **Syntax:** `{rdbms_keepalive_interval, IntervalSeconds}.`
+    * **Example:** `{rdbms_keepalive_interval, 30}.`
     * **Default:** `undefined`
 
-* **odbc_server_type** (local)
-    * **Description:** Specifies RDBMS type. Some modules may optimise queries for certain DBs (e.g. `mod_mam_odbc_user` uses different query for `mssql`).
-    * **Syntax:** `{odbc_server_type, Type}`
+* **rdbms_server_type** (local)
+    * **Description:** Specifies RDBMS type. Some modules may optimise queries for certain DBs (e.g. `mod_mam_rdbms_user` uses different query for `mssql`).
+    * **Syntax:** `{rdbms_server_type, Type}`
     * **Supported values:** `mssql`, `pgsql` or `undefined`
     * **Default:** `undefined`
 
 ### MySQL and PostgreSQL SSL connection setup
 
-In order to establish a secure connection with a database additional options must be passed in aforementioned `odbc_server` tuple.
+In order to establish a secure connection with a database additional options must be passed in aforementioned `rdbms_server` tuple.
```

Comment:
In order to establish a secure connection with a database, additional options must be passed in the aforementioned `rdbms_server` tuple.